modules/Oculus_module/src/HeadRetargeting.cpp

```diff
@@ -7,30 +7,81 @@
  * @date 2018
  */
 
-#include "HeadRetargeting.hpp"
-#include "Utils.hpp"
+// iDynTree
+#include <iDynTree/Core/EigenHelpers.h>
+#include <iDynTree/yarp/YARPConversions.h>
+#include <iDynTree/yarp/YARPEigenConversions.h>
 
-bool HeadRetargeting::configure(const yarp::os::Searchable& config)
+#include <HeadRetargeting.hpp>
+#include <Utils.hpp>
+
+void HeadRetargeting::inverseKinematics(const iDynTree::Rotation& matrix, double& neckPitch,
+                                        double& neckRoll, double& neckYaw)
+{
+    // YXZ decomposition
+    if (matrix(1, 2) < 1)
+    {
+        if (matrix(1, 2) > -1)
+        {
+            neckRoll = std::asin(-matrix(1, 2));
+            neckPitch = std::atan2(matrix(0, 2), matrix(2, 2));
+            neckYaw = std::atan2(matrix(1, 0), matrix(1, 1));
+        } else
+        {
+            neckRoll = M_PI / 2;
```

Comment:
You use `M_PI`, but this will create a failure in compiling in Windows because (as far as I can see) `_USE_MATH_DEFINES` is not defined for this compilation unit, see https://github.com/robotology/QA/issues/81 . 
I suggest  to either add `target_compile_definitions(${EXE_TARGET_NAME} PRIVATE _USE_MATH_DEFINES)`, or even better avoid the use of M_PI.

Reply:
I could use `iDynTree::rad2deg(90)` what do you think?

Reply:
Ops :smile:

Reply:
Done in 50102ab